spark_plugin.py

```diff
@@ -183,7 +185,7 @@ def rest_request(self, url, path, *args, **kwargs):
             if HTTP_404_ERROR not in str(e):
                 collectd.info("Unable to make request at (%s) %s" % (e, url))
             return None
-        except:
+        except Exception:
```

Comment:
Wouldn't somebody want to know about this if there is some unexpected exception that isn't related to HTTP?